DEV - 1/FibonacciNumbers/FibonacciNumbers/Program.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+
+namespace FibonacciNumbers
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter your number");
+            int InputNumber = int.Parse(Console.ReadLine());
```

Comment:
int.Parse can throw exception, you have to process this situation (please read about try-catch statements)